R/run_pacta.R

```diff
@@ -0,0 +1,57 @@
+#' Run PACTA for all portfolios matching a pattern in an input directory
+#'
+#' @param path String. Directory where portfolios are located.
+#' @param regext String. Regular expression that defines a valid portfolio name.
+#'
+#' @examples
+#' run_pacta()
+#' @noRd
+run_pacta <- function(path = "../input", regexp = "_Input[.]csv") {
```

Comment:
TODO: In README, explain that input `.csv` and `.yml` pairs MUST be formatted as: 
- `*_Input.csv` and `*_Input_PortfolioParameters.yml`

TODO: Explain that, within `*_Input_PortfolioParameters.yml`:
- `portfolio_name_in`, `investor_name_in` can be whatever you like, it will populate the output files with columns `portfolio_name` and `investor_name` with the values provided